src/guides/v2.2/design-styleguide/errors-and-messages/errors-and-messages.md

```diff
@@ -2,42 +2,37 @@
 group: admin-style-guide
 subgroup: Writing and Content
 title: Admin Errors and Messages Writing
-menu_title: Errors and Messages Writing
-menu_order: 2
-menu_node:
 ---
 Magento uses different types of dynamic messaging to alert users when:
 
-*  An error has occurred and blocks them in completing a task
+*  An error has occurred and blocks them from completing a task.
 
-*  Users need to know important information before continuing a task
+*  Users need to know important information before continuing a task.
 
-## Types of Errors and Messages
+## Types of errors and messages
 
-*  **Information information.** This messaging is not an error. It's used when a person can continue with the task but needs to be alerted regarding information critical to completing a task.
+*  **Informational**: This messaging is not an error. It is used when a person can continue with the task but needs to be alerted regarding information critical to completing the task.
 
-*  **Recoverable error.** Used when a person is blocked and can only continue by taking an action to fix the problem.
+*  **Recoverable error**: This is used when a person is blocked and can only continue by taking an action to fix the problem.
 
-*  **Not recoverable / catastrophic error.** Used when a person can’t complete the task currently. Usually, it’s due to system error, the server is down, etc. For ideal next steps for users, include a contact method to fix the issue. If a contact method isn't available, provide guidance for the user to try again at a later time.
+*  **Not recoverable/catastrophic error**: Used when a person cannot complete the task. Usually, it is due to a system error: the server is down, etc. Ideally, include a method to fix the issue or provide a contact for resolution. If a method or contact is available, provide guidance for the user to try again at a later time.
 
-## How to Write Errors Content
+## How to write error content
 
 Use complete sentences and brief, exact, but conversational language.
 
-1. Avoid the following words – personal pronouns, "please", and "sorry".
+1. Avoid pleading or personal pronouns. For example: "you, "please", or "sorry".
 
 1. Specify what went wrong.
 
-1. Tell users how to fix the problem and continue the current task. If the problem can’t be resolved online, give users appropriate options to contact us or try again at a later time.
+1. Tell users how to fix the problem and continue the current task. If the problem cannot be resolved online, give users appropriate options to contact us or try again at a later time.
```

Comment:
I wouldn't assume a merchant should always contact Magento, because 3rd party extensions might be supported only by the company who developed it. So maybe change to "give users appropriate workaround options or support information".

Reply:
Done.